Sources/ArcGISToolkit/Documentation.docc/Tutorials/TableTopSceneViewTutorial.tutorial

```diff
@@ -0,0 +1,44 @@
+@Tutorial(time: 10) {
+    @Intro(title: "TableTopSceneView Tutorial") {
+        `TableTopSceneView` provides an augmented reality table top experience.
+        @Image(source: TableTopSceneView, alt: "An image of a table top AR experience.")
+    }
+    
+    @Section(title: "Using the TableTopSceneView") {
+        @ContentAndMedia {
+            @Image(source: TableTopSceneView, alt: "An image of a table top AR experience.")
+        }
+        
+        @Steps {
+            @Step {
+                Initialize a `TableTopSceneView`. Specify the anchor point, translation factor,
+                and clipping distance. 
+                
+                The anchor point is the location point of the ArcGIS Scene that is anchored on a 
+                physical surface. The translation factor determines how many meters the scene view
+                translates as the device moves in the AR experience. The clipping distance is the 
+                distance in meters that the ArcGIS Scene data will be clipped to.
+                @Code(name: "TableTopExampleView.swift", file: TableTopSceneViewStep1)
+            }
+            
+            @Step {
+                Add a `SceneView` to the `TableTopSceneView` closure. The closure builds and overlays 
+                the scene view on top of the augmented reality video feed. Specify the `scene` state 
+                variable.
+                @Code(name: "TableTopExampleView.swift", file: TableTopSceneViewStep2)
+            }
+            
+            @Step {
+                Set the `scene.baseSurface.navigationConstraint` to `.unconstrained` to enable viewing 
+                the scene from below. Set the `scene.baseSurface.opacity` to zero to make the basemap transparent.
+                @Code(name: "TableTopExampleView.swift", file: TableTopSceneViewStep3)
+            }
+            
+            @Step {
+                Use the `SceneView.onSingleTapGesture` view modifer and scene view proxy to perform an 
+                identify operation to identify geoElements at the tapped screen point with a given tolerance.
```

Comment:
It looks like "geoElements" should either be `GeoElement`s or geo elements but I'm not sure which.

Reply:
"geo elements" looks good. Thanks.